website/package.json

```diff
@@ -19,6 +19,7 @@
     "@docusaurus/preset-classic": "3.5.2",
     "@mdx-js/react": "^3.0.0",
     "clsx": "^2.0.0",
+    "docusaurus": "^1.14.7",
     "docusaurus-lunr-search": "^3.4.0",
```

Comment:
Also, I think we can update this to `^3.5.0`. The lunr search isn't technically even hooked up yet (but will be at some point). We might even remove this until it is?

Reply:
Fixed the dependencies.